src/main/java/com/github/piedpiper/node/stepfunctions/StepFunctionsExecute.java

```diff
@@ -0,0 +1,63 @@
+package com.github.piedpiper.node.stepfunctions;
+
+import com.github.piedpiper.node.NodeOutput;
+
+
+import java.util.Optional;
+
+
+import com.amazonaws.services.stepfunctions.model.StartExecutionRequest;
+import com.amazonaws.services.stepfunctions.model.StartExecutionResult;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.commons.utils.JsonUtils;
+import com.github.piedpiper.common.PiedPiperConstants;
+import com.github.piedpiper.node.NodeInput;
+import com.github.piedpiper.node.ParameterMetadata;
+import com.github.piedpiper.utils.ParameterUtils;
+
+
+public class StepFunctionsExecute extends StepFunctionsBaseHandler {
```

Comment:
Again shouldn't be an extension of Base handler. It should be injected via a factors and implement Functions